website/yarn.lock

```diff
@@ -4,45 +4,45 @@
 
 "@algolia/autocomplete-core@1.5.2":
   version "1.5.2"
-  resolved "https://registry.yarnpkg.com/@algolia/autocomplete-core/-/autocomplete-core-1.5.2.tgz#ec0178e07b44fd74a057728ac157291b26cecf37"
```

Comment:
could you please move changes to this file to separate PR which updates deps?

Reply:
Surprisingly when I just simply updated the docs from the master branch there seem to be no changes.

Reply:
So plz remove this file from here :)

Reply:
Done